### ISSUE TITLE
MO2 does not launch in standalone mode.

### ISSUE THREAD

**Issue by Proventius** [sonic2kk/steamtinkerlaunch#976]
## System Information

- SteamTinkerLaunch version: **steamtinkerlaunch-v12.12**
- Distribution: **Nobara**
- Installation Method: **Used 'dnf install steamtinkerlaunch', this installed the noarch version.**

## Issue Description

I typed 'steamtinkerlaunch mo2 start' it installs mo2 but when it goes to start it throws out some output about the nice limit being below a certain number and then it creates a minidump that I'm not sure how to find. After the minidump the application does not appear to start. I have followed the troubleshooting steps outlined [here](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Mod-Organizer-2). I believe that output is inside one of the files attached to this post. I would like to include that I am relatively new to linux and probably need help finding any additional information you might need.

## Logs
[steamtinkerlaunch.log](https://github.com/sonic2kk/steamtinkerlaunch/files/13358992/steamtinkerlaunch.log)
[startMO2_.log](https://github.com/sonic2kk/steamtinkerlaunch/files/13358996/startMO2_.log)

**Comment by sonic2kk** [comment]
You're using a very out of date version of SteamTinkerLaunch, this issue is fixed in SteamTinkerLaunch-git, which I have no idea if it is available in the Nobara repos. You can install with `sudo make install` though, provided you have all the dependencies and `jq` installed.

The main problem as told by the fact that you get a minidump is that the latest MO2 pre-release is being downloaded. This is not compatible with Wine, the minidump output is a result of Wine complaining. This was already investigated in #973, and a duplicate of this problem was already opened in #974. Please search for open issues and [look at the changelog](https://github.com/sonic2kk/steamtinkerlaunch/wiki/Changelog) before opening issues (the issue template already told you to do both of these), and do not use SteamTinkerLaunch v12.12.

Keep in mind as well that because the latest pre-release does not work, this means you cannot mod newer games like Starfield using MO2. The program is simply not compatible with Wine yet, but there is a PR upstream at Wine which implements the missing Qt6 features from Wine causing the crash. Once this Wine patch is merged (hopefully before MO2 v2.5.0 is out in stable), MO2 v2.5.0 will work under Wine.

**Comment by Proventius** [comment]
Alright thanks for the info and sorry.

**Comment by sonic2kk** [comment]
No problem, just be sure to check to make sure the issue isn't already resolved first in future :-) For this project and others as well.

**Comment by Proventius** [comment]
Will do!
